content/posts/2025/05/how-cldt-is-shaping-our-approach.md

```diff
@@ -0,0 +1,48 @@
+---  
+layout: page  
+authors: ["Pre-Seeds Team"]  
+teaser: "The Pre-Seeds team participated in Part One of The Carpentries' Collaborative Lesson Development Training, and here is their recap!"  
+title: "Lessons on Lessons: How The Carpentries' Collaborative Lesson Development Training (CLDT) is Shaping our Approach"  
+Date: 2025-05-12  
+time: "09:00:00"  
+tags: ["CLDT", "Curriculum"]  
+---
+
+_This blog post was first published on [Deborah Udoh's personal blog site](https://medium.com/@deborahudoh02/lessons-on-lessons-how-cldt-is-shaping-our-approach-c16be44fae19), and again on the [Software Sustainability Institute's news and blogs hub](https://www.software.ac.uk/blog/lessons-lessons-how-cldt-shaping-our-approach). It is republished here with only slight formating variations to ensure conformity with The Carpentries blog posts style guide._ 
+
+---
+
+On the 25th to 28th of March, the **Pre-Seeds team participated in Part One of The Carpentries' [Collaborative Lesson Development Training](https://carpentries.org/lesson-development/)** (CLDT). It was an enriching experience, and we are deeply grateful to our facilitators, [Toby Hodges](https://www.linkedin.com/in/toby-hodges-b572382b3/), [Aleksandra Nenadic](https://www.linkedin.com/in/aleksandra-nenadic-bb64b96/), and the larger Carpentries leadership team. We learned a lot and want to share our reflections on what worked well, what could be improved, and how this training will shape our approach moving forward.
+
+## What Worked Well?
+
+One of the standout aspects of the training was the **simplicity and clarity** of the sections. The lessons were structured in a way that was easy to follow, with a great balance of theoretical concepts and hands-on practice. The “show and tell” approach made it easier to understand and retain information, ensuring that learning was interactive rather than passive.
+
+![Screenshot of a Zoom meeting where an instructor is teaching from a colourful slide showing “The life cycle of a lesson.”](/blog/2025/05/lesson-life-cycle-screenshot.jpg)
+
+
+The training was **highly interactive**, with numerous breakout sessions that **encouraged teamwork** and application of new knowledge. These sessions allowed us to collaborate effectively, refine our lesson drafts, and exchange ideas. We also appreciated the facilitators’ commitment to **continuous feedback** — at every step, they checked in to ensure we understood the content and had space to ask questions.
+
+A particularly memorable part of the training was the structured approach to lesson development. We learned to start with preliminary considerations, such as identifying our target audience, understanding their prerequisite knowledge, and pinpointing knowledge gaps. From there, we moved into crafting clear learning objectives and exercises, and using storytelling to make lessons more engaging.

```

Comment:
```suggestion
A particularly memorable part of the training was the structured approach to lesson development. We learned to start with preliminary considerations, such as identifying our target audience, understanding their prerequisite knowledge, and pinpointing knowledge gaps. From there, we moved into crafting clear learning objectives and exercises and using storytelling to make lessons more engaging.
```

Reply:
@OscarSiba I did not want to change the blog post too much from the original, soI  made two suggestions.

Reply:
Thanks, Angelique! 😍